include/util.h

```diff
@@ -29,6 +29,28 @@ namespace connect
     std::string toString(const Events& events);
     std::string toString(const ObjectStream& objectStream);
 
+    inline Status makeSuccess(int code = Status::SUCCESS, int facility = Status::FACILITY_NONE)
```

Comment:
probably makeStatus & makeError functions will be often used, so they better to keep together with class Status (same header)